common/clock/time_source.go

```diff
@@ -22,58 +22,44 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
+// Package timex provides extensions to the [time] package.
```

Comment:
That's not the name of the package?  I see a lot of `timex` packages out there.. if this is based on one of those, why not import it directly? Or if the code is substantially based on one of those, do we need to attribute it?

Reply:
Typo from old code